### DISCUSSION TITLE
info Enocean

### DISCUSSION THREAD

**Discussion by Olodumdomotico** [indomus/forum#2734] [GENERALE - Discussioni generiche]
Sapete se tutti i dispositivi Enocean usano stesso protocollo?? potrei usare pulsante enocean Nodon e attivare relè Eltako etc...?? (tanto per esempio)
HomeAssistant può simulare i pulsanti battery-free (della linea Enocean) in modo da sfruttare  questi relè anche nella domotica???
Grazie

**Comment by NicKGNT** [comment]
I think not.

**Comment by nuitari83it** [comment]
Dunque, ho fatto un po di ricerche avendo anche io switch eltako.
Ho installato sul mio raspberry dove gira HA una chiavetta USB per leggere e mandare segnali sulle frequenze di enocean. 
https://www.enocean.com/de/produkt/usb-300/ 

Considera che in europa gira sugli 868 - 902 e 928 MHz, in america la frequenza é diversa.
Con HA ora vedo le stringhe di protocollo quando avviene una trasmissione, ma devo capire chi é cosa e come triggerare manualmente ON-OFF states.
